fsharp-backend/tests/Tests/Execution.Tests.fs

```diff
@@ -151,7 +151,7 @@ let testListLiterals : Test =
 
 
 let testRecursionInEditor : Test =
-  testTask "results in recursion" {
+  testTask "results in recursion" { // WHATISTHIS
```

Comment:
What does this test do? It's not clear to me.
[full test](https://github.com/darklang/dark/blob/bb6a46f32644f0cd33437e4ee5146248054403f8/fsharp-backend/tests/Tests/Execution.Tests.fs#L154-L192) for reference

Reply:
This tests that we don't execute infinite recursion in the editor, and instead break quickly. 

It's supposed to test this bit of code: https://github.com/darklang/dark/blob/bb6a46f32644f0cd33437e4ee5146248054403f8/fsharp-backend/src/LibExecution/Interpreter.fs#L604